e2e/test/scenarios/embedding/embedding-snippets.cy.spec.js

```diff
@@ -100,6 +108,13 @@ features.forEach(feature => {
                 downloads: false,
               }),
             );
+
+          highlightedTexts().should("have.length", 1);
+
+          cy.findByRole("tab", { name: "Parameters" }).click();
+          cy.findByRole("tab", { name: "Look and Feel" }).click();
+
+          highlightedTexts().should("have.length", 1);
```

Comment:
A small comment might be helpful, I wouldn't know why we're asserting this if I haven't seen Kamil's video recordings